app/searchApi.rb

```diff
@@ -228,6 +228,11 @@ def aws_encode(params, facetTypes, search_type)
   aws_params[:start] = (search_type == "items") ?
         params.dig('start', 0) ? params['start'][0] : 0
      :  params.dig('info_start', 0) ? params['info_start'][0] : 0
+  if aws_params[:query] =~ %r{\b(title:|author:|keyword:|keywords:)}
```

Comment:
Ah, I was wondering how the new parser gets triggered. This is nice and conservative.